dataset/src/main/scala/frameless/ops/GroupByOps.scala

```diff
@@ -173,4 +185,52 @@ class GroupedBy2Ops[K1, K2, V](
   def flatMapGroups[U: TypedEncoder](f: ((K1, K2), Iterator[V]) => TraversableOnce[U]): TypedDataset[U] = {
     underlying.flatMapGroups(f)
   }
+
+  def pivot[P: CatalystPivotable, Values <: HList](pivotColumn: TypedColumn[V, P],
+                                values: Values)(
+    implicit validValues: MatchesAll[Values, P]
+  ): Pivot[V, TypedColumn[V,K1] :: TypedColumn[V, K2] :: HNil, P, Values] =
+    Pivot(self, g1 :: g2 :: HNil, pivotColumn, values)
 }
+
+/** Represents a typed Pivot operation.
```

Comment:
Oh sorry didn't see your text in the PR. I think it would still be nice to have that info in a scaladoc here. I will continue the review with that info